frontend/pages/labels/EditLabelPage/EditLabelPage.tsx

```diff
@@ -49,21 +51,18 @@ const EditLabelPage = ({ routeParams, router }: IEditLabelPageProps) => {
     }
   );
 
-  // TODO: clean this up when API allows getting hosts by
-  // host ids in a single request. We need to make another request when
-  // the label is manual to get the host data for the targeted hosts.
   const {
     data: targetedHosts,
     isLoading: isLoadingHosts,
     isError: isErrorHosts,
-  } = useQuery<{ host: IHost }[], AxiosError, IHost[]>(
+  } = useQuery<IGetHostsInLabelResponse, AxiosError, IHost[]>(
     ["hosts"],
     () => {
-      return hostAPI.getHosts(label?.host_ids ?? []);
```

Comment:
Can the temporary `getHosts` method be removed from `/services/hosts` as well in this PR?

Reply:
Yes, done!  

This would probably be better off as a filter on the "get hosts" API rather than a separate "get hosts for label" API but this is the path of least resistance.